### PR TITLE
added october newsletter draft

### PR DESCRIPTION
**Pull request by kelly-o**
please review 😬

### CODE REVIEW

File: docs/blog/newsletter-october-2017.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+   :tags: newsletter
+
+
+#####################################
+Write the Docs Newsletter – October
+#####################################
+
+Hello friends and fellow documentarians!
+
+Kelly O. here, happy to be arriving in your inbox for another round-up of docs-wisdom gleaned from our community. It's been a big month, since last we spoke. We had our fourth European conference, we've surpassed 2500 members in Slack, AND it's officially been a year since we started this here newsletter experiment. Happy newsletterversary!
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2017.rst]: Remove comma between "month" and "since"

Comment (hillaryfraley) [reply docs/blog/newsletter-october-2017.rst]: Also, love "newsletterversary!"

File: docs/blog/newsletter-october-2017.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+*************************************************
+Putting our URLs to work – for us and our readers
+*************************************************
+How do we make our URLs meaningful to our readers? What should we do when content inevitably moves (or is removed)? How can we make sure all of the great content we write remains discoverable?
+
+When discussing these questions this month, the community started from a point of agreement: good URL paths should carry meaning. For example, a lot of doc sites have URLs that reflect their content hierarchy, like ``site/category/article-title``. Ideally, in such situations, if a reader truncates the path to ``site/category/`` they should find a list of every article in the category.But content restructuring and obsolescence are facts of life for most documentarians. It's unrealistic to think that `Cool URIs don't change <https://www.w3.org/Provider/Style/URI>`_. When the inevitable does happen, it's important to make it as easy as possible for readers to find new locations. That means redirects!
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2017.rst]: Need a space between "category." and "But"

File: docs/blog/newsletter-october-2017.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Creating redirects can be difficult to manage when there are lots of external links when lots of URLs are changing. Sadly, most documentation tools – with the notable exception of MindTouch – don't offer automated redirects. One suggested approach was to use analytics to prioritize your most critical URLs, setting up manual redirects for those pages first to keep your content discoverable. For long term URL hygiene, folks suggested automated link-checking tools to find track down and update broken links and stray pages.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2017.rst]: "...when there are lots of external links when lots of URLs are changing." - Do you want to delete the "when lots of URLs are changing" part?

File: docs/blog/newsletter-october-2017.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+*************************
+The true meaning of "API"
+*************************
+What exactly are we referring to when we say "API"? While the initialism itself stands for Application Programming Interface, it's a more complicated question than it seems. When it came up this month, it sparked a lively conversation that suggested that API might mean different things to different people in different contexts.
+
+Most of the conversation revolved around that idea that, when we say "API" we assume that our reader (or listener) will inherently understand the type of API we're talking about. But that might not be the case as often as you think. If you work with REST APIs, it's likely you'll assume that API has a silent "REST" in front of it. But if you've spent your career working with library APIs, that will probably be the meaning you default to.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2017.rst]: Remove comma between "that" and "when"

File: docs/blog/newsletter-october-2017.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+******************************************
+Proofreading and copyediting your own work
+******************************************
+If you're a lone writer, you've probably been the unenviable position of not having an editor around to proofread, copyedit, or otherwise sanity-check your writing. It's a perennial dilemma, and one that surfaces regularly on Slack. This month produced a particularly good batch of suggestions and resources. To help you clean up your copy when your only editor is you, here's some tips and tricks from the Write the Docs hivemind:
+
+* Put your writing away and review it at least a day later.
+* Print it out on paper and read/edit in the new format.
+* Read each sentence independently. Try reversing the order of sentences, starting with the last one and working your way backward to the first. Or reverse the order of paragraphs.
+* Listen to it being read aloud, either by you or by a device. Text-to-Speech tools are widely available.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2017.rst]: Maybe replace "it" with "your writing" here?

File: docs/blog/newsletter-october-2017.rst [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+* Folks in these roles are generally expected to be the "face" of their developer community. Their work involves making their product/language/framework/etc. more visible and more approachable.
+* Often they sit at an overlap point between engineering – since they're (ideally) experts in the tool they're representing – and sales, marketing, documentation, etc.
+* The line between "evangelist" and "advocate" is blurry, but often the evangelist side is about bringing the product out to the community while the advocate side focuses on bring the community's voice back to the product team.
+* Although these roles may have gotten their start in the open source community, they're becoming increasingly common across the industry.
+
+For further reading on the topic, this medium post has lots of good thoghts: `https://medium.com/google-developers/the-core-competencies-of-developer-relations-f3e1c04c0f5b <https://medium.com/google-developers/the-core-competencies-of-developer-relations-f3e1c04c0f5b>`_
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2017.rst]: "Medium" (need caps)

File: docs/blog/newsletter-october-2017.rst [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+*************************
+Upcoming community events
+*************************
+
+**WRITE THE DOCS DAY AUSTRALIA COUNTDOWN: Six Weeks Left!**
+We're so excited to be coming up on our first official event down under! If you're going to be in the Melbourne area in November, come a full day of sprints, talks, and community-building! Check it out (and get your ticket!) here: `http://www.writethedocs.org/conf/au/2017/ <http://www.writethedocs.org/conf/au/2017/>`_
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2017.rst]: Should it be "...come for a full day..."?

File: docs/blog/newsletter-october-2017.rst [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+Most of the conversation revolved around that idea that, when we say "API" we assume that our reader (or listener) will inherently understand the type of API we're talking about. But that might not be the case as often as you think. If you work with REST APIs, it's likely you'll assume that API has a silent "REST" in front of it. But if you've spent your career working with library APIs, that will probably be the meaning you default to.
+
+There was some question of whether any of this mattered. Whether it's a library, REST, SOAP, or other type of API, they're all still "methods of communication between various software components" (thank you `wikipedia <https://en.wikipedia.org/wiki/Application_programming_interface>`_). But as evidenced by the length and depth of the conversation, it's definitely a topic worth clarifying. Handily enough, one of our Prague speakers, Garen Torikian discusses has a slide on this very topic in his presentation, "Finally! Trustworthy and Sensible API Documentation with GraphQL." Stay tuned for the video recording!
```
Comment (clairealyssa) [inline docs/blog/newsletter-october-2017.rst]: "Garen Torikian discusses has a slide on this very topic" - We should strike "discusses".

File: docs/blog/newsletter-october-2017.rst (hunk 3 above)
Comment (clairealyssa) [inline docs/blog/newsletter-october-2017.rst]: We have "to find" and "track down" in the last sentence here. We should pick one.

Comment (clairealyssa) [reply docs/blog/newsletter-october-2017.rst]: I personally like track down more than to find. :)

File: docs/blog/newsletter-october-2017.rst (hunk 8 above)
Comment (clairealyssa) [reply docs/blog/newsletter-october-2017.rst]: We should also have a comma after Garen Torikian.

File: docs/blog/newsletter-october-2017.rst [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+************************************************
+Defining developer relations/evangelism/advocacy
+************************************************
+This month, someone asked a question which many of us have wondered about, at one point or another: "What does a developer advocate or evangelist actually do?" (You may have even wondered this _as_ a devleoper advocate/evangelist...)
+
+The ensuing conversation made it clear that whether you call it developer relations (DevRel), developer evangelism, or developer advocacy, it can mean a lot of different things. But some common themes emerged:
+
+* Folks in these roles are generally expected to be the "face" of their developer community. Their work involves making their product/language/framework/etc. more visible and more approachable.
+* Often they sit at an overlap point between engineering – since they're (ideally) experts in the tool they're representing – and sales, marketing, documentation, etc.
+* The line between "evangelist" and "advocate" is blurry, but often the evangelist side is about bringing the product out to the community while the advocate side focuses on bring the community's voice back to the product team.
```
Comment (clairealyssa) [inline docs/blog/newsletter-october-2017.rst]: I would break this sentence up - it's a little clunky, if well put.

**Review by clairealyssa — CHANGES_REQUESTED** [review]
Here's my feedback. Thanks for getting this together, Kelly!

File: docs/blog/newsletter-october-2017.rst [hunk 10]
```diff
@@ -0,0 +1,81 @@
+.. post:: October 9, 2017
+   :tags: newsletter
```
Comment (ericholscher) [inline docs/blog/newsletter-october-2017.rst]: I'd love to see tags based on the content of the newsletter, so we can make things easier to find in the future. Like: https://github.com/writethedocs/www/blob/master/docs/blog/newsletter-february-2017.rst

File: docs/blog/newsletter-october-2017.rst [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+
+#####################################
+Write the Docs Newsletter – October
```
Comment (ericholscher) [inline docs/blog/newsletter-october-2017.rst]: I like including the year in the title for consistency.

**Review by ericholscher — COMMENTED** [review]
Small nits

File: docs/blog/newsletter-october-2017.rst (hunk 1 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: done and thanks! ;)

File: docs/blog/newsletter-october-2017.rst (hunk 2 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: 👍

File: docs/blog/newsletter-october-2017.rst (hunk 3 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: ah, just needed an 'or'

File: docs/blog/newsletter-october-2017.rst (hunk 4 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: 👍

File: docs/blog/newsletter-october-2017.rst (hunk 5 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: 👍

File: docs/blog/newsletter-october-2017.rst (hunk 6 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: 👍

File: docs/blog/newsletter-october-2017.rst (hunk 7 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: it certainly should

File: docs/blog/newsletter-october-2017.rst (hunk 3 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: ah, excellent catch!

File: docs/blog/newsletter-october-2017.rst (hunk 8 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: 👍

File: docs/blog/newsletter-october-2017.rst (hunk 9 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: totally – good catch

File: docs/blog/newsletter-october-2017.rst (hunk 10 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: Ooh, yeah. I can dig it.

File: docs/blog/newsletter-october-2017.rst (hunk 11 above)
Comment (kelly-o) [reply docs/blog/newsletter-october-2017.rst]: 👍

**Pull request merged**